pkg/config/analysis/local/istiod_analyze.go

```diff
@@ -306,11 +306,11 @@ func (sa *IstiodAnalyzer) AddRunningKubeSource(c kubelib.Client) {
 }
 
 func isIstioConfigMap(obj any) bool {
```

Comment:
Pilot = collection.NewSchemasBuilder(). does not include cm

Reply:
@hzxuzhonghu Analysis controller is using `All` instead of `Pilot`, as it needs to analyze things like service, ns...

see https://github.com/istio/istio/blob/master/pkg/config/legacy/util/kuberesource/resources.go#L29